test/server_callbacks.c

```diff
@@ -263,13 +266,27 @@ typedef struct {
     void *cbdata;
 } release_cbdata;
 
-static void release_cb(pmix_status_t status, void *cbdata)
+
+static void * _release_cb(void *arg)
 {
-    release_cbdata *cb = (release_cbdata*)cbdata;
+    release_cbdata *cb = (release_cbdata*)arg;
     if (NULL != cb->cbfunc) {
         cb->cbfunc(cb->status, "foobar", cb->cbdata);
     }
     free(cb);
+    spawn_wait = false;
```

Comment:
but here `spawn_wait` is set to `false`. This will work but doesn't seem consistent.